socorro/tests/external/boto/test_crashstorage.py

```diff
@@ -301,8 +299,8 @@ def test_get_dumps_not_found(self, boto_helper):
         with pytest.raises(CrashIDNotFound):
             boto_s3_store.get_dumps("0bba929f-dead-dead-dead-a43c20071027")
 
-    def test_get_dumps_as_files(self, boto_helper, tmpdir):
-        boto_s3_store = self.get_s3_store(tmpdir=tmpdir)
+    def test_get_dumps_as_files(self, boto_helper, tmp_path):
```

Comment:
This switches from the old `tmpdir` pytest fixture which returned a `py.something.Path` to the new `tmp_path` pytest fixture which returns a `pathlib.Path`.